daemon/src/maker_inc_connections.rs

```diff
@@ -354,32 +357,26 @@ impl Actor {
                 }
             },
             RollOverProtocol { order_id, msg } => {
-                if self
+                let _ = self
                     .rollover_actors
                     .send(&order_id, rollover_maker::ProtocolMsg(msg))
-                    .await
-                    .is_err()
-                {
-                    tracing::warn!(%order_id, "No active rollover actor")
-                }
+                    .await;
             }
             Settlement {
                 order_id,
                 msg: taker_to_maker::Settlement::Initiate { sig_taker },
             } => {
-                if self
+                let _ = self
                     .settlement_actors
                     .send(&order_id, collab_settlement_maker::Initiated { sig_taker })
-                    .await
-                    .is_err()
-                {
-                    tracing::warn!(%order_id, "No active settlement");
-                }
+                    .await;
             }
```

Comment:
Sorry to bring this up only now but this change is kind of unintended. The (ephemeral) actors being disconnected here is very much possible which is why I would like to keep these messages.

Reply:
Ahh, my bad. I thought I didn't mess this up.